indico/modules/events/abstracts/forms.py

```diff
@@ -221,3 +224,18 @@ def __init__(self, *args, **kwargs):
             inject_validators(self, 'submitted_for_tracks', [DataRequired()])
         super(MultiTrackMixin, self).__init__(*args, **kwargs)
         self.submitted_for_tracks.query = Track.query.with_parent(event).order_by(Track.title)
+
+
+class AbstractsScheduleForm(IndicoForm):
+    start_dt = IndicoDateTimeField(_("Start"), default_time=time(0, 0),
+                                   description=_("The moment users can start submitting abstracts"))
+    end_dt = IndicoDateTimeField(_("End"), [LinkedDateTime('start_dt')], default_time=time(23, 59),
+                                 description=_("The moment the submission process will end"))
+    modification_end_dt = IndicoDateTimeField(_("Modification end"), [Optional(), LinkedDateTime('end_dt')],
+                                              default_time=time(23, 59),
+                                              description=_("The submitted abstracts cannot be modified after this "
```

Comment:
Yes, it's technically the same since the date has to be after the `end_dt` (we do check that, right?). But the idea behind the feature is not so much to disallow editing after date X but rather to allow editing until date X even though the submission period already ended at date Y (<X)